tools/python/build_route_api.py

```diff
@@ -0,0 +1,85 @@
+import urllib.request
```

Comment:
Please format this script according to PEP8. pylint tool may help you in it. We usually try to fix all reasonable pylint recommendations.